test-forms/wired_checkout.html

```diff
@@ -20,7 +20,7 @@
 										.prechecked{background-color:#fff; text-align: center; font-size: 16px;}
                                     </style>
                                     <div id="isgiftholder" style="pointer-events: none;">
-									<div id="isGiftCheck" aria-labelledby="isGiftCheckText" data-isgift="false" role="button" tabindex="0"><img id="nextgen-gift-check" src="https://subscribe.wired.com/circulation/wired/images/ng-radio-check-new.png" alt=""></div>
+									<div id="isGiftCheck" aria-labelledby="isGiftCheckText" data-isgift="false" role="button" tabindex="0"></div>
```

Comment:
One thing I realised while looking into this is that we didn't remove images or scripts from these forms. Ivy did remove them when collecting the forms with the scraper, but we didn't do it for the other forms. Probably a nice win to remove everything from all these examples so tests run faster. Something for another time.